pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -780,3 +780,111 @@ func TestFitScore(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkTestFitScore(b *testing.B) {
+	tests := []struct {
+		name                 string
+		requestedPod         *v1.Pod
+		nodes                []*v1.Node
+		existingPods         []*v1.Pod
+		expectedPriorities   framework.NodeScoreList
+		nodeResourcesFitArgs config.NodeResourcesFitArgs
+	}{
+		{
+			name: "1000nodes/RequestedToCapacityRatio",
+			requestedPod: st.MakePod().
+				Req(map[v1.ResourceName]string{"cpu": "3000", "memory": "5000"}).
+				Obj(),
+			nodes: []*v1.Node{
+				st.MakeNode().Name("node1").Capacity(map[v1.ResourceName]string{"cpu": "4000", "memory": "10000"}).Obj(),
+			},
+			existingPods: []*v1.Pod{
```

Comment:
I think we can also remove the `existingPods` from the struct, build it inside the `b.Run()` directly.